content/docs/command-reference/params/index.md

```diff
@@ -145,51 +146,38 @@ $ dvc run -n train -d logs/ -o users.csv \
 
 ## Examples: Python parameters file
 
-Consider this parameters file in Python format, named `params.py`:
+Consider this Python parameters file named `params.py`:
 
 ```python
-IS_BOOL: bool = True
-CONST = 5
-
-# All standard variable types are supported
-FLOAT = 0.001
+BOOL = True
+INT = 5
 STR = 'abc'
-DICT = {
-    "a": 1,
-    "b": 2
-}
-LIST = [1, 2, 3]
-SET = {4, 5, 6}
-TUPLE = (10, 100)
-NONE = None
-
-
-# It is possible to retrieve either class constants
-# or own variables defined in __init__
+DICT = {'a': 1, 'b': 2}
+# etc.
+
+# DVC can retrieve class constants and variables defined in __init__
 class TrainConfig:
+
     EPOCHS = 70
 
     def __init__(self):
-        # TrainConfig.layers param will be 9
         self.layers = 5
-        self.layers = 9
-        # TrainConfig.foo will NOT be found because the complex expression
-        self.foo = 1 + 2
-        # TrainConfig.bar will NOT be found
-        bar = 1
```

Comment:
same - why change this?

Reply:
Went from 7 lines to 3, looks better IMO and has the same info.

If you refer to removing the `bar = 1` line and its comment above, again we're explaining how Python works there — the variable has local scope so it can't be accessed externally (unless I'm wrong).

Reply:
I think in this case all of this was beneficial, you see at once pretty much different example and see what might work what not

Reply:
OK. I added `bar` back, PTAL.